homeassistant/components/slack/notify.py

```diff
@@ -193,14 +193,22 @@ async def _async_send_remote_file_message(
 
     async def _async_send_text_only_message(self, targets, message, title, blocks):
         """Send a text-only message."""
+        username = self._username
+        icon = self._icon
+        icon_type = (
+            "url" if self._icon.lower().startswith(("http://", "https://")) else "emoji"
```

Comment:
We generally don't want complex expressions if they wrap lines. I would simplify:

```python
if self._icon.lower().startswith(("http://", "https://")):
    icon_type = "url"
else:
    icon_type = "emoji"
```